packages/mongodb-log-writer/src/mongo-log-manager.ts

```diff
@@ -34,9 +36,20 @@ export class MongoLogManager {
   _options: MongoLogOptions;
 
   constructor(options: MongoLogOptions) {
+    if (options.prefix) {
+      if (!/^[a-z0-9_]+$/i.test(options.prefix)) {
```

Comment:
nit: Merging these two `if`s would make this a bit easier to read imo 🙂 

```suggestion
    if (options.prefix && !/^[a-z0-9_]+$/i.test(options.prefix)) {
```